src/components/Nav/NavTop.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FaRegHeart } from 'react-icons/fa';
+import { FaUserCircle } from 'react-icons/fa';
+import { FaChevronDown } from 'react-icons/fa';
+import { FaSearch } from 'react-icons/fa';
+
+function NavTop() {
+  return (
+    <Header>
+      <Nav>
+        <WrapInfo>
+          <Logo>IOI</Logo>
+          <Subscribe>구독</Subscribe>
+          <Store>스토어</Store>
+          <WrapSearch>
+            <Input />
+            <FaSearch size="13" />
+          </WrapSearch>
+        </WrapInfo>
+        <WrapUser>
+          <CreatorCenter>크리에이터 센터</CreatorCenter>
+          <FaRegHeart size="22" className="icon-like" />
+          <WrapProfile>
+            <FaUserCircle size="28" className="icon-user" />
+            <FaChevronDown size="15" />
+          </WrapProfile>
+        </WrapUser>
+      </Nav>
+    </Header>
+  );
+}
+
+const Header = styled.header`
+  position: fixed;
+  padding: 20px 0;
+  width: 100vw;
+`;
+
+const Nav = styled.div`
+  ${props => props.theme.variables.flex('row', 'space-between', 'center')}
+  margin: 0 auto;
+  width: 1140px;
+`;
+
+const WrapInfo = styled.div`
+  ${props => props.theme.variables.flex('row', 'center', 'center')}
+`;
+
+const Logo = styled.h1`
+  margin: 0 35px 0 0;
+  display: inline-block;
+  font-size: 30px;
+  font-weight: bold;
+`;
+
+const Subscribe = styled.span`
+  margin: 0 15px 0 0;
+  font-size: 20px;
+  font-weight: bold;
+  cursor: pointer;
+`;
+
+const Store = styled(Subscribe)`
+  margin: 0 30px 0 0;
+`;
+
+const WrapSearch = styled.div`
+  padding: 10px 20px;
+  width: 380px;
+  border-radius: 4px;
+  background-color: #f8f8f8;
+  cursor: pointer;
+`;
+
+const Input = styled.input.attrs({
+  placeholder: '찾으시는 취미가 있나요?',
+})`
```

Comment:
```suggestion
const Input = styled.input`
```

위의 리뷰를 적용해 보면 이 부분은 제거 되어도 되겠습니다!